Meadowlark-js/packages/meadowlark-authz-server/src/model/SigningKey.ts

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+import memoize from 'fast-memoize';
+
+function getSigningKey(): Buffer {
+  const signingKeyEncoded = process.env.SIGNING_KEY;
+  if (signingKeyEncoded == null) {
+    throw new Error('Must have a base-64 encoded signing key. Try creating a new one with `yarn createKey`');
+  }
+  return Buffer.from(signingKeyEncoded, 'base64');
+}
+
+export const signingKey = memoize(getSigningKey);
```

Comment:
I've heard you mention memoize before, but it isn't really in my consciousness yet. I had to study this a bit to understand it. Should I have used this in the environment variable cache work?

Isn't the signing key needed on the core meadowlark API as well, for validating a token? So would it make sense to have this in a shared utility library, maybe in the Environment module? 

I agree about not using the existing `getStringFromEnvironment` function, because it is useful to have this custom error message. Or could add custom error message as an an optional argument on that method.

Reply:
I used memoize because we transform the environment variable into a Buffer before providing it, so this just prevents doing that every time.
I think we are moving token validation exclusively to the auth server in RND-336.

Reply:
That's what I get for reviewing during a meeting - forgetting the entire point of this! Treat Meadowlark API as a client of the OAuth server. It asks the OAuth server for verification, and it does not need to have the signing key.